fix: skip Alive =false backend before add job.

wangwq10 opened the pull request:
Fist Doris cluster works fine even if lost some be.
In this case cluster have 5 BEs and one of them Alive = false, and replica all reallocated
It should be able to create a ccr job
![backend](https://github.com/user-attachments/assets/bf531598-f7f3-4cf8-bcb1-99e00c8a49c0)

Commit:
fix: skip Alive =false backend before add job.

w41ter commented:
@wangwq10 That's a valid point, but the objective is to verify whether the enable_feature_binlog option of all BEs is enabled, if we skip any BE, the check constraint might be broken: some backends don't enable the binlog so the syncer could not download the binlog files when the backend becomes active again, and the replication will be stopped.

wangwq10 committed:
Merge branch 'selectdb:dev' into dev

wangwq10 committed:
Update job_manager.go

fix: after pause update successfully, status/state should align to  job.State and return "paused" as excepted.
